Updated contributing tab in Readme! (#942)
hasura/graphql-engine: README.md

[![Docs](https://img.shields.io/badge/docs-v1.0-brightgreen.svg?style=flat)](https://docs.hasura.io)

> [![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)

Reviewer: Please add these two lines back.

Author: Resolved the issue!

If you want to report a security issue, please [read this](SECURITY.md).

## Contributing 

> We :heart: contributions from everyone! We welcome [bug reports](https://github.com/hasura/graphql-engine/issues/), [bug fixes](https://github.com/hasura/graphql-engine/pulls) and feedback on the library is always appreciated. Look at the [Contributor Guidelines](CONTRIBUTING.md) for more details.

Reviewer: ```suggestion
We :heart: contributions from everyone! Check out our [contributing guide](CONTRIBUTING.md) for more details.
```

Author: Resolved the issue!

> ## [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/hasura/graphql-engine/issues) [![GitHub contributors](https://img.shields.io/github/contributors/hasura/graphql-engine.svg)](https://GitHub.com/hasura/graphql-engine/graphs/contributors/)

Reviewer: ```suggestion
```
